<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISAJ</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/Sources/icon.png</normaloff>:/img/Sources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="1">
     <widget class="QWidget" name="cameraWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>250</width>
        <height>380</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="camerasetWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QWidget" name="dipRoot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>210</width>
              <height>140</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>410</width>
              <height>370</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 0, 0);</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QVideoWidget" name="cameraDisp" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cameralistBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="scaninfoWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="idSt">
            <property name="text">
             <string>学号：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLine"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameSt">
            <property name="text">
             <string>姓名：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameLine"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="cameraCtrlWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="cameraOpen">
            <property name="text">
             <string>开启</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cameraClose">
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ScreenShot">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>截图</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="dispWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="searchWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLine"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchBtn">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="tableWidget" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QTableWidget" name="table">
            <property name="styleSheet">
             <string notr="true">selection-background-color: rgba(255, 255, 127, 227);
selection-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>加载</string>
     </property>
     <addaction name="actionDefaultOpen"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="menu_2"/>
    <addaction name="actionNewTable"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>添加</string>
     </property>
     <addaction name="actionRecode"/>
     <addaction name="actionRow"/>
     <addaction name="actionColumn"/>
    </widget>
    <addaction name="menu"/>
   </widget>
   <widget class="QMenu" name="OCR">
    <property name="title">
     <string>OCR</string>
    </property>
    <widget class="QMenu" name="menuBaidu">
     <property name="title">
      <string>Baidu</string>
     </property>
     <widget class="QMenu" name="menuHandWrite">
      <property name="title">
       <string>手写</string>
      </property>
      <addaction name="actionuseHandWrite"/>
      <addaction name="actionSetIdKeyWord"/>
     </widget>
     <addaction name="actionSetting"/>
     <addaction name="separator"/>
     <addaction name="menuHandWrite"/>
    </widget>
    <widget class="QMenu" name="menuOC">
     <property name="title">
      <string>OC搜索</string>
     </property>
     <addaction name="actionAsID"/>
     <addaction name="actionAsname"/>
    </widget>
    <addaction name="menuBaidu"/>
    <addaction name="menuOC"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="OCR"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="iconText">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionRecode">
   <property name="text">
    <string>一次记录</string>
   </property>
  </action>
  <action name="actionRow">
   <property name="text">
    <string>一行</string>
   </property>
  </action>
  <action name="actionColumn">
   <property name="text">
    <string>一列</string>
   </property>
  </action>
  <action name="actionDefaultOpen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>默认打开上次保存的文件</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionuseHandWrite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>使用手写</string>
   </property>
  </action>
  <action name="actionSetIdKeyWord">
   <property name="text">
    <string>设置学号关键字</string>
   </property>
  </action>
  <action name="actionAsID">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>通过学号</string>
   </property>
  </action>
  <action name="actionAsname">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>通过名字</string>
   </property>
  </action>
  <action name="actionNewTable">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
